src/pages/markdown.css

```diff
@@ -1,3 +1,7 @@
+.markdown {
+  max-height: 100svh;
```

Comment:
For some reason, putting `max-h-svh` in the HTML file did not work, despite tailwing documentation saying it should be supported. So this works for now